GMG/Scenes/ChordProgress/Segment.swift

```diff
@@ -186,8 +176,46 @@ struct Segment: View {
         }
     }
 
+    struct ChordCandidateButton: View {
+        let index: Int
+        let chord: Chord
+        let selectedChord: Chord?
+        let onTap: () -> Void
+
+        private var backgroundColor: Color {
+            if chord.description == selectedChord?.description {
```

Comment:
description으로 비교하지 않고 타입 자체로 비교해주세요!

```suggestion
            if selectedChord == chord {
```